lldb/test/API/functionalities/progress_reporting/swift_progress_reporting/TestSwiftProgressReporting.py

```diff
@@ -0,0 +1,67 @@
+"""
+Test that we are able to broadcast and receive swift progress events from lldb
+"""
+import lldb
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test.decorators import *
+import lldbsuite.test.lldbutil as lldbutil
+import threading
+
+class TestSwiftProgressReporting(TestBase):
+
+    mydir = TestBase.compute_mydir(__file__)
+
+    eBroadcastBitStopProgressThread = (1 << 0)
```

Comment:
is this temporary & should it be declared in the SB API?

Reply:
thanks, I guess the name threw me off. Maybe it doesn't need the `e` prefix since it's not from an enum (that is what the `e` stands for right?)